articles/connections/_otp-limitations.md

```diff
@@ -0,0 +1,3 @@
+* Only the last one-time password issued will be accepted. Once the latest one is issued, any others are invalidated.
```

Comment:
is `password` a right word for a passwordless connection documentation?

Reply:
I mean, by itself, no, but in the context of the whole phrase `one-time password` - that's literally what it is. However, we could reword it to say something like "only the most recently issued code will be accepted" if you like that better?

Reply:
looks like we are using the word `password` in a lot of places (even in the dashboard), so I will leave the decision to the product team

Reply:
We call this a "one-time-use code" throughout the rest of the passwordless docs. Just FYI.

Reply:
I feel like that's a thing the PE team/Product needs to discuss. OTP is an industry-standard term. As far as I know, it hasn't been changed in any existing (not new) docs, or as @siacomuzzi says, in the dashboard.

Reply:
Consistency across the board is what we should go for, but IMO it doesn't need to be accomplished via this PR.